packages/block-editor/src/hooks/behaviors.js

```diff
@@ -30,73 +90,28 @@ import merge from 'deepmerge';
  */
 export const withBehaviors = createHigherOrderComponent( ( BlockEdit ) => {
 	return ( props ) => {
+		const blockEdit = <BlockEdit key="edit" { ...props } />;
```

Comment:
This addressed @youknowriad's feedback from the original PR - #https://github.com/WordPress/gutenberg/pull/49972#discussion_r1203612193.